packages/core/src/routes/connector.ts

```diff
@@ -125,6 +123,29 @@ export default function connectorRoutes<T extends AuthedRouter>(
     }
   );
 
+  router.get(
+    '/connectors/:target/:platform',
```

Comment:
nit: not sure whether this route will conflict with other route or may look confuse.